chapter_convolutional-neural-networks/channels.md

```diff
@@ -77,7 +77,14 @@ import torch
 ```
 
 ```{.python .input}
-%%tab mxnet, pytorch
+%%tab jax
+from d2l import jax as d2l
+import jax
+from jax import numpy as jnp
+```
+
+```{.python .input}
+%%tab mxnet, pytorch, jax
 def corr2d_multi_in(X, K):
     # Iterate through the 0th dimension (channel) of K first, then add them up
     return sum(d2l.corr2d(x, k) for x, k in zip(X, K))
```

Comment:
This line requires function saved in #2316, so we can re-run the CI once that PR is merged.